kiwi/system/root_bind.py

```diff
@@ -88,9 +88,12 @@ def mount_kernel_file_systems(self):
         """
         try:
             for location in self.bind_locations:
-                if os.path.exists(location):
+                mount_point = os.path.normpath(os.sep.join([
```

Comment:
I'm not sure under which circumstances one of self.bind_locations does not exist in the image root directory but I'm ok with the extra check. For a better understanding what mount_point is could we use a more expressive name, e.g

```python
location_mount_target = os.path.normpath(...
```

Thoughts ?